### ISSUE TITLE
[Accessibility Issue] [Survey Designer][Property Pane]: remove option from choices in property pane doesn't read it as remove button

### ISSUE THREAD

**Issue by dmitrykurmanov** [surveyjs/survey-creator#869]
based on https://surveyjs.answerdesk.io/internal/ticket/details/T4556
